doc/release/README.prereqs

```diff
@@ -24,3 +24,18 @@ about your environment for using Chapel:
   code using compilers by Cray, IBM, Intel, and PGI. Any standard C and
   C++ compiler should work, but the aforementioned are the ones we have
   the most experience with.  
+
+
+We have used the following commands to set up newly installed system images.
+Note that our test system works much better with tcsh installed and that users
+of Chapel won't need 'git'.:
```

Comment:
I don't think git should be included here. I think developer requirements should be kept separate as they are more complicated (e.g. a developer probably needs/wants gcc 4.7+, python 2.7, flex, bison, gdb, valgrind, etc). Maybe they can be listed somewhere in doc/developer/ ?